Pipfile

```diff
@@ -4,11 +4,12 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-pytest = "~=6.0.1"
-pytest-mock = "~=3.3.1"
-pre-commit = "~=2.15.0"
-aws-sam-cli = "~=1.35.0"
-awscli = "~=1.22.5"
+pytest = "~=6.0"
+pytest-mock = "~=3.3"
+pre-commit = "~=2.15"
+aws-sam-cli = "~=1.35"
+awscli = "~=1.22"
+coverage = "~=7.0"
```

Comment:
i don't think this project needs `autopep8`?

Reply:
good catch, I'm not even sure why it was here to begin with